js/tests/visual/tooltip.html

```diff
@@ -57,6 +62,21 @@ <h1>Tooltip <small>Bootstrap Visual Test</small></h1>
 
     <script>
       $(function () {
+        if (typeof document.body.attachShadow === 'function') {
+          var shadowRoot = $('#shadow')[0].attachShadow({ mode: 'open' })
+          shadowRoot.innerHTML = [
+            '<button type="button" class="btn btn-secondary" data-toggle="tooltip" data-placement="top" title="Tooltip on top in a shadow dom">',
+              'Tooltip on top in a shadow dom',
+            '</button>',
+            '<button id="secondTooltip" type="button" class="btn btn-secondary" data-toggle="tooltip" data-placement="top" title="Tooltip on top in a shadow dom with container option">',
+              'Tooltip on top in a shadow dom',
+            '</button>'
```

Comment:
Any reason you use an Array instead of a string?

Reply:
For me it's more readable but that's a matter of preference